lib/ansible/modules/cloud/amazon/elb_target_group.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: elb_target_group
+short_description: Manage a target group for an Application load balancer
+description:
+    - "Manage an AWS Application Elastic Load Balancer target group. See \
+    U(http://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html) for details."
+version_added: "2.4"
+author: "Rob White (@wimnat)"
+options:
+  name:
+    description:
+      - The name of the target group.
+    required: true
+  protocol:
+    description:
+      - The protocol to use for routing traffic to the targets. Required if state=present.
+    required: false
+    choices: [ 'http', 'https' ]
+  port:
+    description:
+      - "The port on which the targets receive traffic. This port is used unless you specify a port override when registering the target. Required if \
```

Comment:
Can we wrap the line properly, rather than using `\`

```
  description:
  - The port on which the targets receive traffic. This port is used unless you specify a port override when registering 
    the target. Required if I(state) is C(present)
```